cmd/juno/cli/root.go

```diff
@@ -153,6 +270,27 @@ func init() {
 		"config file (default is %s)", filepath.Join(config.Dir, "juno.yaml")))
 	rootCmd.PersistentFlags().StringVar(&dataDir, "dataDir", "", fmt.Sprintf(
 		"data path (default is %s)", config.DataDir))
+	// RPC
+	rootCmd.PersistentFlags().StringP("rpcport", "p", os.Getenv("RPCPORT"), "Set the RPC Port")
```

Comment:
Any issue using `Flags()` instead `PersistentFlags()`? I think is better to use `Flags()` if is possible

Reply:
Done.